packages/modules/manage-versions/src/components/CreateUpdateVersion/VersionModal.test.tsx

```diff
@@ -72,7 +72,7 @@ describe("VersionModal", () => {
     ).toBe(2);
 
     const actionButton = document.querySelector(
-      ".iui-button.iui-high-visibility"
+      "[data-iui-variant=high-visibility]"
```

Comment:
Can you see if we can use the action text button instead of the class ? 
`screen.getByText("Action");` should do the same.

Reply:
Action wasn't working so replace with test ids